devel-common/src/tests_common/test_utils/config.py

```diff
@@ -85,3 +85,60 @@ def env_vars(overrides):
             os.environ[env] = value
         for env in new_vars:
             os.environ.pop(env)
+
+
+@contextlib.contextmanager
+def task_sdk_conf_vars(overrides):
```

Comment:
Why do we need this?

Reply:
It's the task sdk variant of `conf_vars` context manager (uses `from airflow.sdk.configuration import conf` in comparison to `conf_vars` which uses: `from airflow.configuration import conf` and it also doesnt call `settings.configure_vars` which basically set up:

```
    global SQL_ALCHEMY_CONN
    global SQL_ALCHEMY_CONN_ASYNC
    global DAGS_FOLDER
    global PLUGINS_FOLDER
    global DONOT_MODIFY_HANDLERS
```

We use this fixture in tests which are non core -- sdk tests, providers etc. Check example usage in `task-sdk/tests/task_sdk/bases/test_hook.py` for example.